PDP supports videos OOTB (#741)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/product-details/containers/product-gallery.mdx

    ['imageParams', 'object', 'No', 'Parameters for resolving image URLs.'],
    ['thumbnailParams', 'object', 'No', 'Parameters for resolving thumbnail URLs.'],

>     ['zoom', 'object or boolean', 'No', 'Configuration for the zoom feature. If an object, it can have a closeButton property to show a close button. Defaults to false.'],

Reviewer: ```suggestion
    ['zoom', 'boolean or object', 'No', 'Configuration for the zoom feature. If an object, it can have a closeButton property to show a close button. Defaults to false.'],
```

Reviewer: Not a part of your PR, but I like placing `boolean` first, before `object` (like you wrote it). Changing here for consistency.

    ['videos', 'boolean or object', 'No', 'Configuration for video support in the gallery. Set to true to enable videos (positioned after images) or pass an object with a position property. Defaults to false (disabled).'],
    ['scope', 'string', 'No', 'Unique identifier for the PDP context. Only containers rendered with this scope will respond to product events.'],
  ]}
/>

## Videos

> The `videos` prop enables product video support in the gallery carousel. Videos are disabled by default.

Reviewer: ```suggestion
The `videos` prop enables product video support in the gallery carousel. The following usage example shows the different configuration options: disabled (default), enabled, and placing your videos before or after your static images within the carousel.
```

### Usage

```js
// Disabled (default) — backward compatible
<ProductGallery />
<ProductGallery videos={false} />

// Enabled — videos appear after images
<ProductGallery videos />
<ProductGallery videos={true} />

// Videos appear before images
<ProductGallery videos={{ position: 'first' }} />

// Videos appear after images (explicit)
<ProductGallery videos={{ position: 'last' }} />
```

### Supported video sources

> The gallery supports multiple video sources, rendered differently depending on the URL type:

Reviewer: ```suggestion
The gallery supports multiple video sources and formats, rendering each based on its URL type.
```

<OptionsTable
  compact
  options={[
    ['Video source', 'Render method'],
    ['Direct files (.mp4, .webm, .ogg, .mov, .avi, .mkv)', 'Native <video> element with playback controls'],
    ['Other URLs (YouTube, Vimeo, etc.)', '<iframe> with embedded player'],
  ]}
/>

> 

Reviewer: Copy and paste the table below to use the latest `TableWrapper` control with rewording for clarity (I think!)

<TableWrapper>

| Video source | Render method |
|--------------|---------------|
| Video file URLs — URLs that resolve directly to a video file (.mp4, .webm, .ogg, .mov, .avi, .mkv) | Native `<video>` element with playback controls |
| Video platform URLs (YouTube, Vimeo, etc.) | `<iframe>` with embedded player |

</TableWrapper>

## Example

> The following example demonstrates how to configure the `ProductGallery` container with video support:

Reviewer: ```suggestion
The following example demonstrates how to configure the `ProductGallery` container with video support.
```

> ```js

Reviewer: ```suggestion
```js {18}
```

Reviewer: This is an expressive code feature that highlights lines in code examples. You can also use it for multiple lines of code {1-6} as well as non-adjacent lines {1-6, 9, 16}
